Documentation/ceph-object-bucket-claim-crd.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Object Bucket Claim CRD
+weight: 2800
+indent: true
+---
+
+# Ceph Object Bucket Claim CRD
+
+Rook allows creation and customization of object bucket claim through the custom resource definitions (CRDs). The following settings are available for Ceph Object Bucket Claim. It also requires storage class
```

Comment:
"object bucket claims" -- plural.

Reply:
Change first paragraph to:
" Rook supports the creation of  buckets via an Object Bucket Claim OBC) which is a custom resource described by the object bucket custom resource definition (CRD)."